### DISCUSSION TITLE
XHTTP3 似乎没有实现 QUIC 0-RTT？

### DISCUSSION THREAD

**Discussion by xqzr** [XTLS/Xray-core#4093] [Ideas]
客户端
```json
{
                              "tlsSettings": {
					"enableSessionResumption": true,
					"alpn": [
						"h3"
					]
				}
}
```
Nginx 也已经 `ssl_early_data on;`
抓包未见 0-RTT。记得以前的 QUIC “传输方式”抓包可见 0-RTT

**Comment by RPRX** [comment]
没研究过 QUIC 0-RTT，如果跟 TLS 那个一样则不够安全，况且 QUIC 的复用也是 0-RTT，又没有队头堵塞

**Reply by xqzr** [reply]
> QUIC 的复用也是 0-RTT，又没有队头堵塞

虽然如此，但是我经常看见 QUIC 握手
